### ISSUE TITLE
Resource and Storage Manager

### ISSUE THREAD

**Issue by eiacopini** [GeoNode/geonode#10167]
Hi i had installed geonode v. 4.0.0post1
Does this version allow you to upload data that is stored in external storage such as dropbox? if so how?
Thank you :)

**Comment by giohappy** [comment]
yes and no @eiacopini. Everything is in place to support it but we still miss an interface to drive it (API and UI). 
For the API it shouldn't be a big deal. It should be a matter of extending its serializer.

**Comment by eiacopini** [comment]
Thank you @giohappy for clarification. Does the same situation is also valid for the latest version 4.0.1?

**Comment by giohappy** [comment]
We don't have plans to work on it. Maybe someone else?

Il mer 19 ott 2022, 23:12 Eleonora Iacopini ***@***.***> ha
scritto:

**Comment by gannebamm** [comment]
The Thünen-Institute will do some work on the API v2 and eg. enable document uploads via API in the upcoming months. This could be the first part of getting a dropbox/file share ingestion system working.

**Comment by giohappy** [comment]
@gannebamm good to know! You know that the storage manager already has support for remote files r/w, thanks to the smart-open library. Let's try to converge on a common approach ;)

**Comment by eiacopini** [comment]
Thank you at all for the good news:)

**Comment by giohappy** [comment]
Closing as part of a repository cleanup: this issue was last updated before 2025-01-01. Not a judgment on the report itself — feel free to reopen or file a fresh issue against `main` if it's still relevant. Thanks for the report.
